Jenkinsfile

```diff
@@ -84,57 +74,65 @@ String gcpLabel(String s) {
     s.replaceAll(/[^a-zA-Z0-9_-]+/, '-').toLowerCase().take(62)
 }
 
-def runIntegrationTest(String description, String kubeprodArgs, String ginkgoArgs, Closure clusterSetup, Closure dnsSetup) {
-    timeout(120) {
-        // Regex of tests that are temporarily skipped.  Empty-string
-        // to run everything.  Include pointers to tracking issues.
-        def skip = ''
+def runIntegrationTest(String description, String kubeprodArgs, String ginkgoArgs, boolean pauseForDebugging, Closure clusterSetup, Closure dnsSetup) {
+    // Regex of tests that are temporarily skipped.  Empty-string
+    // to run everything.  Include pointers to tracking issues.
+    def skip = ''
 
-        withEnv(["KUBECONFIG=${env.WORKSPACE}/.kubeconf"]) {
-            clusterSetup()
+    clusterSetup()
 
-            withEnv(["PATH+KTOOL=${tool 'kubectl'}"]) {
-                sh "kubectl version; kubectl cluster-info"
+    waitForRollout("kube-system", 30)
 
-                unstash 'binary'
-                unstash 'manifests'
-                unstash 'tests'
-
-                sh "kubectl --namespace kube-system get po,deploy,svc,ing"
+    container('kubectl') {
+        sh "kubectl version"
+        sh "kubectl cluster-info"
+    }
 
-                sh "./bin/kubeprod -v=1 install ${kubeprodArgs} --manifests=manifests"
+    retry(3) {
```

Comment:
This will probably help quite a lot, but makes me sad.  Whenever this retry kicks in, we're basically hiding a problem from ourselves that will be encountered by users (unlike retries for dns/platform issues).

Reply:
I'm also in favor of moving retries to kubeprod/kubecfg instead of hiding the real problem.

Reply:
sounds good!